system/Common.php

```diff
@@ -1121,15 +1125,17 @@ function stringify_attributes($attributes, bool $js = false): string
      * returns its return value if any.
      * Otherwise will start or stop the timer intelligently.
      *
+     * @param non-empty-string|null    $name
      * @param (callable(): mixed)|null $callable
      *
-     * @return Timer
+     * @return mixed|Timer
+     * @phpstan-return ($callable is (callable(): mixed) ? mixed : Timer)
```

Comment:
I think this should be based on `$name`?

Reply:
See the code. `stop()` and `start()` return `$this`.
https://github.com/codeigniter4/CodeIgniter4/blob/64866aa3c46a4e9bfc484e9da850b14329865e7f/system/Common.php#L1134

Reply:
Okay. Then based on `$name` and `$callable`.

Reply:
Oh, you are correct. Fixed.